browser/src/slideshow/SlideShowPresenter.ts

```diff
@@ -669,76 +669,90 @@ class SlideShowPresenter {
 			this._hideSlideControls.bind(this),
 			3000,
 		);
-	}
+	};
+
+	private _createSlideButton(
```

Comment:
formatting looks off, did you run checks?

`make prettier` will show you the errors
`make prettier-write` will auto correct formatting to required format